contracts/sophia.md

```diff
@@ -9,6 +10,73 @@ to a blockchain (the Æternity BlockChain). Thus some features of conventional
 languages, such as floating point arithmetic, are not present in Sophia, and
 some blockchain specific primitives, constructions and types have been added.
 
+## Table of Contents
```

Comment:
Is this autogenerated or will this have to be updated with every new addition? Who's in charge for keeping track of this?

Reply:
^^ maybe the script to generate it can go to top level?

Reply:
Not autogenerated. A better overview was much in demand. Whoever is adding something to the documentation could expand the table of contents as well?

Reply:
History shows stuff like this will be terribly out to date after a few iterations, and be more confusing that helpful. So I'd propose to either have a script to generate this or choose an alternative frontend than GitHub for this.

Reply:
As far as I recognize there wasn't any script used at the "State Channels" sections as well. So who could make a script proposal? People from the community are really struggling with a bad overview and long searches.

Reply:
I tend to agree with @Pegahbit that SC already have ToC here: https://github.com/aeternity/protocol/tree/master/channels

Script or different frontend could be additional improvement PR as it seems a decision of maintaining the ToC manually is already taken with SC.

Reply:
I agree, just noticed this, definitely can be a separate improvement

Reply:
So what's the status now? It's still open and not merged. Could we solve this please?